_targets/build.project.armv7m7-imxrt117x

```diff
@@ -63,12 +63,11 @@ PREINIT_SCRIPT=(
 
 # Production user script contains applications to run Phoenix-RTOS
 USER_SCRIPT=(
-	"wait 2000"
 	"kernel ${BOOT_DEVICE}"
 	"app ${BOOT_DEVICE} -x dummyfs xip1 ocram2"
 	"app ${BOOT_DEVICE} -x imxrt-multi ocram2 ocram2"
-	"app ${BOOT_DEVICE} -x psh xip1 ocram2")
-
+	"app ${BOOT_DEVICE} -x psh xip1 ocram2"
+	"wait 2000")
```

Comment:
In this case the `wait 2000` is unnecessary, because the plo will always stop on the command line (wasting 2s more), maybe You thought: 
```suggestion
	"wait 2000"
	"go!")
```

Reply:
That's right. I wanted to add it in the next PR (because some change in test runner is needed for that), but `imxrt117x` CI isn't enabled yet, so I can do it now.